content/post/deeplearning_in_go_part_1.md

```diff
@@ -0,0 +1,393 @@
++++
+title = "Deep Learning from Scratch in Go - Part 1: Equations Are Graphs"
+subtitle = ""
+date = "2017-04-19T08:43:45+10:00"
+latex = true
+draft = true
+
++++
+
+(Author: Chewxy, @chewxy on [Twitter](https://twitter.com/chewxy) and Gophers Slack)
+
+Welcome to the first part of many about writing deep learning algorithms in Go. The goal of this series is to go from having no knowledge at all to implementing a latest paper. 
```

Comment:
Maybe "... having no knowledge at all to implementing some of the latest developments in this area."